bulk.lua

```diff
@@ -163,13 +163,17 @@ local function is_acceptable_item(item_name)
   return acceptable_item_cache[item_name]
 end
 
-function M.acceptable_items(inventory, limit)
+function M.acceptable_items(inventories, limit)
+  local items_map = {}
   local out = {}
-  for name in pairs(inventory.get_contents()) do
-    if is_acceptable_item(name) then
-      out[#out+1] = name
-      if #out >= limit then
-        return out
+  for _,inventory in pairs(inventories) do
```

Comment:
[style] space between for-loop bindings for non-numeric for:
`for _, inventory in`